src/Sowp/NewsModuleBundle/Tests/Controller/CollectionControllerTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace NewsModuleBundle\Tests\Controller;
+
+use Doctrine\ORM\EntityManager;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class CollectionControllerTest extends WebTestCase
+{
+    /** @var $em EntityManager */
+    private $em;
+
+    protected function setUp()
+    {
+        self::bootKernel();
+
+        $this->em = static::$kernel->getContainer()
+            ->get('doctrine')
+            ->getManager();
+    }
+
+    public function testIndex()
+    {
+        echo __FUNCTION__."\n";
+        $client = $this->createClient();
+        $crawler = $client->request('GET', '/kolekcje/');
+
+        $this->assertEquals(200, $client->getResponse()->getStatusCode(),
+            'Request should return HTTP 200');
+
+        $this->assertEquals(1,
+            $crawler->filter('html:contains("Collection list")')->count(),
+            "HTML doesn't contain proper heading");
+    }
+
+    public function testCollectionStructure()
+    {
+        echo __FUNCTION__."\n";
+        $collectionRepository = $this->em->getRepository('NewsModuleBundle:Collection');
+        $slugs = $collectionRepository->getCollectionsSlugs();
```

Comment:
Ok a teraz puściłem doctrine:fixtures:load , mam już nowe slugi..... zmieniać za każdym razem testy?
Kolejna sprawa zwróć uwagę że to także służy do policzenia ile jest kolekcji (oczywiści bardziej poprawnie i elegancko jest mieć w repo jakiś count ale tu jednym wywołaniem załatwiam dwie rzeczy - sprawdzam czy są kolekcje i w razie potrzeby mam slugi dla crawlera....